### ISSUE TITLE
Make requesting method toggleable for Admins

### ISSUE THREAD

**Issue by rotated8** [emory-libraries/blacklight-catalog#850]
This is stretch work for after #849 finishes.

Hyrax uses a gem called [Flipflop](https://github.com/voormedia/flipflop) to allow admins to turn features on and off.

- [x] Add this gem to Blacklight
- [x] Allow only LTDS users to access the UI
- [x] Create a toggle for requesting method
- [x] Configure the Services Page to show if the toggle is off
- [ ] Configure the API methods to show if the toggle is on

**Comment by lovinscari** [comment]
@abelemlih - I have assigned this to you. Please check with @rotated8 on any specific questions. I suspect I will have more issues added to backlog in the coming days but this is something I think we can proceed with now.

**Comment by abelemlih** [comment]
PR ready for review: https://github.com/emory-libraries/blacklight-catalog/pull/1033

**Comment by abelemlih** [comment]
@lovinscari @eporter23 A few items to note regarding this ticket:
- The view that displays requesting using the API does not have a request button, because the code for the button was commented out. I chose to leave it commented, and address it in a separate bug, to avoid expanding the scope of this specific ticket.
- Previously, in the requesting view that uses the API, the policy displayed the policy description of an item for a guest user, and the due date policy for a logged in user. When I was testing the view, I realized that the ALMA API never returns the due date policy for a logged in user, and rather returns "Log in for more information". This is likely due to the fact the code we currently have does not pass user information currently to retrieve the due date policy correctly. For now, I switched the policy value to policy description for both logged in users and guests so the policy value displayed is never "Log in for more information", and in a separate bug ticket we can investigate the exact reason why the due date policy is not retrieved. This is another issue I want to work on separately to avoid expanding the scope of this ticket beyond the feature toggle.

Let me know if you have any questions, and I am happy to make any adjustments if needed.

**Comment by bwatson78** [comment]
Just awaiting admin list.

**Comment by lovinscari** [comment]
@bwatson78 and @abelemlih - I sent the admin list via Slack on 11/11. @bwatson78 - I think you were going to send NetID's so I am not sure if this ticket needs to move from To Accept back to In Progress?

**Comment by lovinscari** [comment]
I confirmed I have access to blackcat-arch.library.emory.edu/features and can see the feature demonstrated by Ayoub. I am not sure if this needs to move to To Deploy to production?

**Comment by bwatson78** [comment]
The code is in Production, but not activated, since the admin list isn't there. The moment you'd like that to go live, I can add the list and redeploy the code, which will turn on the feature for you.
